src/main/java/org/truffleruby/RubyContext.java

```diff
@@ -242,6 +242,11 @@ public Object send(Object object, String methodName, DynamicObject block, Object
                 RubyArguments.pack(null, null, method.getMethod(), DeclarationContext.METHOD, null, object, block, arguments));
     }
 
+    @TruffleBoundary
```

Comment:
Who's calling this from compiled code?

Reply:
Nobody, I followed the style of shutdown(), but indeed both can be removed now.